engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -283,4 +283,54 @@ <h5 translate ng-show="repository.content_type == 'docker'">Upload Docker Image<
     </form>
   </section>
 
+  <section ng-show="repository.content_type === 'yum'">
+    <div class="divider"></div>
+    <h4 translate>Distribution Information</h4>
```

Comment:
Should we just hide this all together if there isn't one for a given repository? And should we call it distribution or kickstart information? I don't know that I know the difference between the two, but it feels like we are doubling up on our terminology given we show 'kickstart repositories' for Red Hat content.